client/src/main/java/io/pravega/client/stream/impl/ReaderGroupImpl.java

```diff
@@ -499,5 +513,6 @@ private String generateSilentCheckpointId() {
     @Override
     public void close() {
         synchronizer.close();
+        ackProcessor.close();
```

Comment:
Why is this line not reached in the test? It looks like the reader group is being closed...

Reply:
ReaderGroupImplTest.java where unit testcases are written for readerGroup was not having readerGroup close() method call. Added shutdown() method which triggers close call on readerGroup.